models/au_abs_community_profiles/code/upload.py

```
@@ -0,0 +1,76 @@
+
+def main() -> None:
+    only = set(a for a in sys.argv[1:]) or None
+    slugs = [
+        Path(p).stem for p in sorted(glob.glob(str(OUTPUT_DIR / "*.parquet")))
+    ]
+    for slug in slugs:
+        if only and slug not in only:
+            continue
+        try:
+            upload_table(slug)
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)